addons/web_editor/controllers/main.py

```diff
@@ -688,20 +730,15 @@ def save_library_media(self, media):
 
         return attachments
 
-    @http.route("/web_editor/get_ice_servers", type='json', auth="user")
-    def get_ice_servers(self):
-        return request.env['mail.ice.server']._get_ice_servers()
+    @http.route("/web_editor/get_editor_collaboration_infos", type='json', auth="user")
+    def get_editor_collaboration_infos(self, model_name, field_name, res_id):
+        signatures = generate_channel_signatures(request.env, model_name, field_name, res_id)
+        return {
+            'ice_servers': request.env['mail.ice.server'].sudo()._get_ice_servers(),
+            'next_expiration_timestamp': signatures['next_expiration_timestamp'],
+            'signatures': signatures['signatures'],
```

Comment:
Si c'est utilisé comme ça autant renvoyer un tuple plutôt qu'un dict.

Reply:
Alternativement,
```python
return {
    'ice_servers': request.env['mail.ice.server'].sudo()._get_ice_servers(),
    **generate_channel_signatures(request.env, model_name, field_name, res_id)
}
```
mais bon...

Reply:
In terms of naming, even though they are technically HMAC digests/signatures, their role is to be (opaque) *channel identifiers*. As the name appears on the wire, it seems more appropriate to name them `channels` or `channel_ids` or something.